src/gui/gui.c

```diff
@@ -127,26 +121,29 @@ int main(void)
     return 0;
 }
 
-void send_message()
+void run_process_generator()
 {
-    int msgq_id = msgget(GUI_SHKEY, 0666 | IPC_CREAT);
-    if (msgq_id == -1)
+    // Get path to the process.out
+    char *args[5];
+    args[0] = "/home/marwan/mywork/Synergify/bin/process_generator.out";
```

Comment:
Try to use the relative path function made in this project for consistency. 
```c
char file_path[PATH_SIZE];
getAbsolutePath(file_path, "process_generator.out");
```
Just know that it requires both executables to be in the `/bin` folder.